app/controllers/books_controller.rb

```diff
@@ -0,0 +1,74 @@
+# Manages the "Book" model
+class BooksController < ApplicationController
```

Comment:
Makes sense! Would we ever want to restrict parts of the non-admin components? One way that was recommended was adding the authorization check in the parent ‘ApplicationController’ then whitelisting either entire controllers or just certain actions within a controller. Restricting everything by default helps prevent accidental access. Regardless, I might push making this decision off until the admin stuff becomes more concretized - I don’t see it causing pains as is.